geode-core/src/test/java/org/apache/geode/cache/client/ClientCacheFactoryJUnitTest.java

```diff
@@ -68,13 +68,14 @@
 @Category({IntegrationTest.class, ClientServerTest.class})
 public class ClientCacheFactoryJUnitTest {
 
-  private ClientCache cc;
+  private ClientCache clientCache;
   private File tmpFile;
 
   @After
   public void tearDown() throws Exception {
-    if (this.cc != null && !this.cc.isClosed()) {
-      cc.close();
+    System.out.println("tearDown");
```

Comment:
Did you want to remove this println before merging?

Reply:
I'll remove that println